CHANGELOG.md

```diff
@@ -1,5 +1,139 @@
+
+    This change pretty-prints JSON received from the Inspector, so JSON output is much easier to read in `wrangler dev`.
+
+    [jspspike]: https://github.com/jspspike
+    [pull/1371]: https://github.com/cloudflare/wrangler/pull/1371
+    [issue/1249]: https://github.com/cloudflare/wrangler/issues/1249
+
+  - **Don't install wasm-pack when publishing a type:webpack project - [EverlastingBugstopper], [issue/745] [pull/1344]**
+
+    We shouldn't install wasm-pack if your project doesn't need it. We thought we fixed this in 1.7.0, but we didn't. This time it's fixed for real, we swear.
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [pull/1344]: https://github.com/cloudflare/wrangler/pull/1344
+    [issue/745]: https://github.com/cloudflare/wrangler/issues/745
+
+  - **Pin wasm-pack and cargo-generate to specific versions instead of always downloading the latest version - [jspspike], [issue/1240] [pull/1358]**
+
+    Previously, every time `wasm-pack` or `cargo-generate` was needed, Wrangler would download and install the latest version. Now, we pin those dependencies to a specific version in every release of Wrangler. If you have either of these tools installed locally, Wrangler will use them if you have a more recent version.
+
+    [jspspike]: https://github.com/jspspike
+    [pull/1358]: https://github.com/cloudflare/wrangler/pull/1358
+    [issue/1240]: https://github.com/cloudflare/wrangler/issues/1240
+
+  - **Support batched KV PUT and DELETE operations to improve Workers Site publish performance - [jspspike], [issue/1191] [pull/1339]**
+
+    Previously, you couldn't use Workers Sites if you wanted to upload more than 10,000 static files, and it took a lot of time to upload close to that many files. This change instead batches upload and delete calls, allowing us to increase the limit and improve performance for everyone.
+
+    [jspspike]: https://github.com/jspspike
+    [pull/1339]: https://github.com/cloudflare/wrangler/pull/1339
+    [issue/1191]: https://github.com/cloudflare/wrangler/issues/1191
+
+- ### Maintenance
+
+  - **Add Code of Conduct - [EverlastingBugstopper], [pull/1346]**
+
+    We actually didn't have one before, so now we do!
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [pull/1346]: https://github.com/cloudflare/wrangler/pull/1346
+
+  - **Deprecate undocumented KV `bucket` attribute - [ashleymichal], [issue/1136] [pull/1355]**
+
+    The `bucket` attribute was not originally intended to be released into the wild. We announced deprecation in March, and are now removing it as an available configuration property.
+
+    [ashleymichal]: https://github.com/ashleymichal
+    [pull/1355]: https://github.com/cloudflare/wrangler/pull/1355
+    [issue/1136]: https://github.com/cloudflare/wrangler/issues/1136
+
+  - **Refactor to separate KV commands, implementation of said commands, and site-specific logic - [ashleymichal], [pull/1332]**
```

Comment:
wonder if we can come up with a better tagline here, not the worst but it's a bit wordy

Reply:
refactor allthethings